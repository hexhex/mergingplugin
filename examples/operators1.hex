pass(0, X) :- &hex["testpred.",""](X).

opanswer(X) :- &operator["testop1", pass, kv](X).
result(Pred, Arity) :- opanswer(X), &answersets[X](AS), &tuples[X, AS](Pred, Arity).
