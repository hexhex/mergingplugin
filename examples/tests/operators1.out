{result(xyz,0), result(foo,0)}
